pom.xml

```diff
@@ -23,7 +23,9 @@
   </issueManagement>
 
   <properties>
-    <sonarlint.core.version>6.5.0.37122</sonarlint.core.version>
+    <maven.compiler.source>11</maven.compiler.source>
```

Comment:
When using the `parent-oss` pom, it is simpler to change the property `jdk.min.version` only.